### PR TITLE
dynamic-counters example

### PR DESCRIPTION
**Pull request by Gregor1971**
| Q                       | A
| ----------------------- | ---
| Fixed Issues?           | example for discussion of #10
| Documentation only PR   | yes
| Patch: Bug Fix?         |
| Minor: New Feature?     |
| Major: Breaking Change? |
| Tests Added + Pass?     |

Adds another example, based on redux-saga/cancellable-counters.

This example demonstrates a dynamic list of dynostore dynamic components.

### CODE REVIEW

File: examples/redux-saga/dynamic-counters/src/App.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Counter from './components/Counter'
+import { ADD_COUNTER } from './actionTypes'
+
+const initialState = { counterIds: ['Counter one', 'Counter two'] }
+
+export const pageReducer = (state = initialState, action) => {
```
Comment (mpeyper) [inline examples/redux-saga/dynamic-counters/src/App.js]: move to the `reducers` folder?

File: examples/redux-saga/dynamic-counters/src/App.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  switch (action.type) {
+    case ADD_COUNTER:
+      return { ...state, counterIds: [...state.counterIds, action.value] }
+    default:
+      return state
+  }
+}
+
+class DynaCounter extends React.Component {
```
Comment (mpeyper) [inline examples/redux-saga/dynamic-counters/src/App.js]: Move to the `compnents` folder.  Could add a link to it directly in the README.

File: examples/redux-saga/dynamic-counters/src/App.js (hunk 1 above)
Comment (mpeyper) [reply examples/redux-saga/dynamic-counters/src/App.js]: Also not sure on the `page` name.  `counters`?

**Comment by mpeyper** [comment]
Great work!

Don't forget to [add yourself as a contributor](https://github.com/ioof-holdings/redux-dynostore/blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) using the "Examples" option.

File: examples/redux-saga/dynamic-counters/src/App.js (hunk 2 above)
Comment (Gregor1971) [reply examples/redux-saga/dynamic-counters/src/App.js]: moved and linked

File: examples/redux-saga/dynamic-counters/src/App.js (hunk 1 above)
Comment (Gregor1971) [reply examples/redux-saga/dynamic-counters/src/App.js]: moved into `reducers/index.js`
un-defaulted previous reducer and renamed to `counterInstanceReducer`
renamed state slice from `page` to `counters`
renamed id array value from `counterIds` to `allIds`

**Pull request merged**